### PR TITLE
login page and routing

### PR DESCRIPTION
**Pull request by daniel-pienaar**

### CODE REVIEW

File: src/main.ts
Comment (Adrian-Hawkins) [inline src/main.ts:1]: Why was this changed?

File: src/app/app.module.ts
Comment (Adrian-Hawkins) [inline src/app/app.module.ts:1]: Do @Stef-DJ Changes work without the module, I don't see the app.component containing any of these imports

**Review by Adrian-Hawkins — CHANGES_REQUESTED** [review]
Please remove all spec files, to generate angular schematics without tests you can use the --skip-tests flag

File: src/app/app.module.ts
Comment (daniel-pienaar) [reply src/app/app.module.ts:1]: yes, i've tested all changes locally, it's part of the migration to standalone components.

File: src/main.ts
Comment (daniel-pienaar) [reply src/main.ts:1]: Part of the migration to standalone components. Changes tested locally, no need to bootstrap using module anymore. I've removed the old commented out code as i forgot to do that

Comment (Adrian-Hawkins) [reply src/main.ts:1]: Just out of curiosity was this changed as to avoid using standalone app.componet?

Comment (daniel-pienaar) [reply src/main.ts:1]: app.component is now standalone and does not need to be included as an ngmodule. I was able to remove app.module with these changes as it wasn't necessary anymore

File: src/styles.scss
Comment (Adrian-Hawkins) [inline src/styles.scss:1]: Out of curiosity how can one refer to these global variables?

**Review by Adrian-Hawkins — APPROVED** [review]

File: src/styles.scss
Comment (daniel-pienaar) [reply src/styles.scss:1]: by using var(--variable-name) in any other css file
